.github/workflows/static_code_checks.yaml

```diff
@@ -0,0 +1,26 @@
+# only has to pass for python 3.8
+name: Static code checks
+
+on:
+  push:
+    branches:
+      main
+  pull_request:
+    branches:
+      main
+
+jobs:
+  run-code-check:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python 3.8
```

Comment:
Doesn't really matter now but should we update this to a Python 3.9 - 3.11 to stay consistent with the pyproject.toml?

Reply:
yes it does! thanks for catching it :) I upgraded it to 3.9 which is the same as FL4Health.